library/core/src/array/mod.rs

```diff
@@ -37,7 +37,7 @@ pub use iter::IntoIter;
 ///
 /// # Example
 ///
-/// Creating muliple copies of a `String`:
+/// Creating multiple copies of a `String`:
```

Comment:
Note that despite my comments about altering compiler behavior, I would be happy to review and approve a PR which only touched the code in the library directory.